patches/dns-stuff.c.patch

```diff
@@ -0,0 +1,32 @@
+diff --git a/dirmngr/dns-stuff.c b/dirmngr/dns-stuff.c
+index 0edbc04..0d0b1ed 100644
+--- a/dirmngr/dns-stuff.c
++++ b/dirmngr/dns-stuff.c
+@@ -117,6 +117,10 @@
+ 
+ #define RESOLV_CONF_NAME "/etc/resolv.conf"
+ 
++#if defined (__MVS__) && !defined(NI_MAXHOST
```

Comment:
```suggestion
+#if defined (__MVS__) && !defined(NI_MAXHOST)
```
Can we remove this with the zoslib change?

Reply:
yes, i have removed. Thank you for review, Igor